components/bio-formats-tools/src/loci/formats/tools/XMLValidate.java

```diff
@@ -70,18 +120,7 @@ public static void main(String[] args) throws Exception {
     }
```

Comment:
Should the call above also use the new `validate` methods (and close the `BufferedReader` internally)?